recipes/uldaq/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "uldaq" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5cfc152a67381717bcf6f275f4f059c7adfae830eb4dfca35d90972a6021f84c
+  patches:
+    - 0001-ul_c_interface-use-ctypes.util.find_library.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - pip
+    - python
+  host:
+    - libuldaq
+    - pip
+    - python
```

```suggestion
  host:
    - pip
    - python
```
We don't need `libuldaq` during the build (since the package does not link to this library and is just a pure Python package without custom install logic).
Also, `requirements: build` should only have build-only tools like compilers etc. Linked libraries and runtimes like Python all go into `requirements: host`.